internal/pkg/describe/uri.go

```diff
@@ -214,43 +214,47 @@ func (d *BackendServiceDescriber) URI(envName string) (URI, error) {
 	}, nil
 }
 
-type albDescriber struct {
-	svc             string
-	env             string
-	svcDescriber    ecsDescriber
-	envDescriber    envDescriber
-	initLBDescriber func(string) (lbDescriber, error)
-	envDNSNameKey   string
+type albOrCdnDescriber struct {
+	svc              string
+	env              string
+	svcDescriber     ecsDescriber
+	envDescriber     envDescriber
+	initLBDescriber  func(string) (lbDescriber, error)
+	albCfnOutputName string // The DNS name for the public or private ALB.
```

Comment:
```suggestion
	albCFNOutputName string // The DNS name for the public or private ALB.
```
https://github.com/golang/go/wiki/CodeReviewComments#initialisms